.github/workflows/release.yaml

```diff
@@ -0,0 +1,35 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.1
+
+      # See https://github.com/helm/chart-releaser
+      # See https://github.com/helm/chart-releaser-action
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@v1.2.1
+        with:
+          config: cr.yaml
+          charts_dir: charts
+        env:
+          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
@TheAlgo Someone with settings access will need to create this.

Reply:
There is no need to create this as it's [provided by GitHub](https://docs.github.com/en/actions/reference/authentication-in-a-workflow).

Reply:
Weird, I swear back in the day I had to make this. Thank you for the link!